rootProject.name = "assertj-automation-root"

include "assertj-error-prone"
throw new RuntimeException('''The excavator failed to run properly - exited with code 1 and output:\n\n''' + file('gradle/exception').text + '\n\n')
